frontend/webapp/views/main/templates/hijack.htm

```diff
@@ -1086,6 +1086,16 @@ <h5 class="modal-title" id="deleteHijacksModalLabel">Delete Hijack</h5>
                 $("#download_table_button").attr("href", url);
             }
 
+            function format ( d ) {
```

Comment:
should this have a more representative name? What is it formatting exactly?

Reply:
I will rename it format_bgp_update_expand